index.html

```diff
@@ -0,0 +1,192 @@
+        integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA=="
+        crossorigin="anonymous" />
+    <!-- Cascading Style Sheet -->
+    <link rel="stylesheet" href="style.css" />
+    <!--------- Boxicon ---------->
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+    <header>
+        <!-- ---- start of top-header ------>
+        <section class="top-header">
+            <div class="container flex">
+                <a href="#"><i class='bx bxl-facebook'></i></a>
+                <a href="#"><i class='bx bxl-twitter'></i></a>
+                <a href="#">English</a>
+                <a href="#">My Page</a>
+                <a href="#">Logout</a>
+            </div>
+        </section>
+
+        <!-------- stat of main-navigation --- -->
+        <nav class="navbar">
+            <div class="container">
+                <div class="nav-logo">
+                    <a href="index.html"><img src="images/nav-img/logo.svg" alt="nav-logo"></a>
+                </div>
+                <ul class="nav-links flex">
+                    <li><a href="about.html" id="active">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Courses</a></li>
+                    <li><a href="#">Contact</a></li>
+                    <button type="submit" class="btn btn-default">
+                        <a href="#">New Courses!</a>
+                    </button>
+                </ul>
+                <i class="open fa fa-bars"></i>
+                <ul class="mobile-nav">
+                    <i class="close fa fa-times"></i>
+                    <li><a href="index.html" id="active">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Courses</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>
+        </nav>
+
+        <!-- ------ start of home section  -->
+        <section class="home">
+            <div class="container">
+                <div class="home-text">
+                    <h2>Welcome! to Evans code online school</h2>
+                    <h1>GROW YOUR CODING SKILLS TO ADVANCE YOUR CAREER PATH</h1>
+                    <p>Our aim to help you a establish a good fundation in software development. The journey
+                        of becoming a developer can not just be achieved by feeling good behind a complter. We make sure
+                        you gain much understanding of every basic lesson in coding.
+                    </p>
+                    <h3>Starting 05.06.23(Mon) ~ (Fri)</h3>
+                    <h6>@ 8am Opo-louker street, West Hills-Weija. Ghana</h6>
+                    <!-- <a href="#" class="hero-btn">Visit us</a> -->
+                </div>
+            </div>
+
+        </section>
```

Comment:
- [ ] Please remove unwanted comments from your codebase because it's your codebase make messy if you want to check your previous code then check your commit GitHub History :+1: 
- [ ] As I mentioned above :point_up: please stick to the Behance design your hero section in the mobile version is not the same as the Behance design :+1: check below :point_down: the comparison between  your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-05-27 00-06-26](https://github.com/evansnyamekye/Capstone-PR/assets/45226848/fa874f02-d0f7-4603-8b7d-792deb7345ba)

**Behance Design**
![Screenshot from 2023-05-27 00-06-11](https://github.com/evansnyamekye/Capstone-PR/assets/45226848/67d1d2cb-643a-41ed-9267-2267b8366807)